plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/fs/DBNFileSystem.java

```diff
@@ -96,7 +96,7 @@ protected void dispose(boolean reflect) {
 
     @Override
     public String getNodeType() {
-        return NodePathType.dbvfs.name() + ".fileSystem";
```

Comment:
`dbvfs.fileSystem ` used in CloudBeaver to determinate the required authentication.